web/pages/css/document.css

```diff
@@ -102,3 +103,37 @@
   font-style: italic;
   text-align: center;
 }
+
+/* Override default modal CSS */
+
+#about_modal .modal-header {
+  color: #fff;
+  background-color: #00457b;
+  border-radius: 0;
+  padding: 8px 12px 7px;
+}
+
```

Comment:
Can we tweak the padding a bit to look more inline with our other dialogs?  Typically we have little to no padding on the outside (right) of the scrollbar.  I achieved this look by setting the following styles:

```
#about_modal .modal-body {
  padding: 0;
}

#about_modal .wv-document {
  padding: 10px 15px 0 10px;
}
```
<img width="527" alt="Screen Shot 2020-04-24 at 1 58 45 PM" src="https://user-images.githubusercontent.com/1480000/80242689-bec11380-8633-11ea-8864-77694b81e567.png">

Reply:
Fixed!